native/packages/react-native-bpk-component-card/stories.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { StyleSheet, View, Platform } from 'react-native';
+import BpkCard from './index';
+import BpkText from '../react-native-bpk-component-text';
+
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const {
+  spacingSm,
+  spacingBase,
+  colorGray50,
+  colorRed500,
+  colorGreen500,
+  colorYellow500,
+  colorBlue700,
+  colorGray500,
+} = tokens;
+
+const styles = StyleSheet.create({
+  centered: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'center',
+    padding: spacingSm,
+    width: '100%',
+    backgroundColor: colorGray50,
+  },
+});
+
+const copy = (
+  <View>
+    <BpkText textStyle="xxl" style={{ color: colorBlue700 }}>Flights to Edinburgh</BpkText>
+    <BpkText textStyle="xl" style={{ color: colorRed500 }}>Flights to Edinburgh</BpkText>
+    <BpkText textStyle="lg" style={{ color: colorGreen500 }}>Flights to Edinburgh</BpkText>
+    <BpkText textStyle="base" style={{ color: colorYellow500 }}>Flights to Edinburgh</BpkText>
+    <BpkText textStyle="sm" style={{ color: colorBlue700 }}>Flights to Edinburgh</BpkText>
+    <BpkText textStyle="xs" style={{ color: colorGray500 }}>Flights to Edinburgh</BpkText>
+  </View>
+);
+
+storiesOf('BpkCard', module)
+  .addDecorator(getStory =>
+    <View style={styles.centered}>
+      {getStory()}
+    </View>,
+)
+  .add('Default', () => (
+    <View>
```

Comment:
We should use the content described in the JIRA card for the docs here so that we can use these stories to generate the appropriate screenshots.

Reply:
Not sure I'm following here, are you saying name the stories based on the different configuration I've got in the JIRA ticket? or something else?

Reply:
Ignore my comment, I've been told what to do 😄